### PR TITLE
Adds a balloon meant to simulate the arrest hud

### PR DESCRIPTION
**Pull request by lordpidey**
## About The Pull Request

Make a balloon that makes you LOOK like you have an arrest warrant out.

## Why It's Good For The Game

Just another toy to add to the arcade stuff.  The gimmick is it will trick security into thinking you have an arrest warrant.

## Changelog
:cl:
add: Arcade toy companies have found a surplus of promotional balloons from an old boyband and put them in the prize rotation.  Warning:  balloons may only be partially inflated, and may incite security forces to arrest you.
/:cl:

### CODE REVIEW

**Comment by trexter555** [comment]
watch as security brings you all the way to the brig anyways

**Comment by ATH1909** [comment]
holy shit based

**Comment by Tharcoonvagh** [comment]
Put it on the other side of the character sprite

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -168,6 +168,14 @@
 		var/mob/M = loc
 		SEND_SIGNAL(M, COMSIG_CLEAR_MOOD_EVENT, "badass_antag", /datum/mood_event/badass_antag)
 	. = ..()
+	
+	
+/obj/item/toy/balloon/arrest
+	name = "arreyst balloon"
```
Comment (Winterous) [inline code/game/objects/items/toys.dm:174]: Typo, "arreyst"

Comment (Jack7D1) [reply code/game/objects/items/toys.dm:174]: It's a feature

Comment (lordpidey) [reply code/game/objects/items/toys.dm:174]: Arreyst was a super popular boyband.

Comment (lordpidey) [reply code/game/objects/items/toys.dm:174]: Changed the description to reference this.

**Review by Twaticus — APPROVED** [review]
baste

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Winterous) [reply code/game/objects/items/toys.dm:174]: Well I certainly have egg on my face.

**Pull request merged**